File: Prak 7/C/PRAK603-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>

int main(){
    int brs1, brs2, i, in, hasil;
    scanf("%d %d", &brs1, &brs2);
    int baris1[brs1], baris2[brs2];
    if (brs1 == brs2){
        for (i = 0; i < brs1; i++){
            scanf("%d", &in);
            baris1[i] = in;}
        for (i = 0; i < brs2; i++){
            scanf("%d", &in);
            baris2[i] = in;}
        for (i = 0; i < brs1; i++){
            hasil = baris1[i] * baris2[i];
            printf("%d ", hasil);}}
    else {
        printf("Jumlah tidak sama");}
return 0;}

File: Prak 7/Python/PRAK603-2210817110005-MIZZUDDINABDISSALAM.py
brs1, brs2 = map(int, input().split())
if brs1 == brs2 :
    angka1 = list(map(int, input().split()))
    angka2 = list(map(int, input().split()))
    baris1 = []
    baris2 = []
    a = 0
    baris1.append(angka1[a : a + brs1])
    baris2.append(angka2[a : a + brs2])
    for i in range(0, brs1) :
        print(baris1[0][i] * baris2[0][i], end=' ')
else :
    print("Jumlah tidak sama")
